2996_reflection/reflection.md

```diff
@@ -4401,6 +4401,10 @@ namespace std::meta {
 
       template<class T> requires constructible_from<string, T>
         consteval name_type(T &&);
+
+      u8string $get-u8string$() const;    // $exposition only$
+      string $get-string$() const;        // $exposition only$
+      bool $is-utf8$() const;             // $exposition only$
```

Comment:
I think it's better to word this as an exposition-only `variant<u8string, string> contents;`

Reply:
Done.